bin/alluxio-fuse.sh

```diff
@@ -13,29 +13,29 @@
 SCRIPT_DIR="$(cd "$(dirname "$(readlink "$0" || echo "$0")")"; pwd)"
 
 get_env () {
-  DEFAULT_LIBEXEC_DIR="${SCRIPT_DIR}"/../../../libexec
+  DEFAULT_LIBEXEC_DIR="${SCRIPT_DIR}"/../libexec
```

Comment:
The purpose is to promote Fuse and make it very easy to use. Put the script at root would give user the impression that it is a first-class citizen.

Reply:
I don't see why `integration/fuse/bin` wouldn't be a first-class citizen. If we need to put all first class citizens in `bin`, we'll end up with too many scripts in that directory.

Reply:
Then how about we keep the fuse script in `integration/fuse/bin/`, but add an entry in `bin/alluxio`?

Reply:
I'm hesitant to add more types of functionality there since the API is already very crowded - @apc999 what are your thoughts about where to organize our scripts?

Reply:
I actually have a different idea: leave this script still in `integration/fuse/bin/`, but in `alluxio-start.sh`, depending on whether `fuse` is enabled, automatically start fuse without asking users to call this script.

Reply:
in my next PR, I allow multiple mounting points of fuse, so there is no default mounting point and user needs to run `mount`

Reply:
Let me move back the script, and we can decide if we want a shortcut from the main script later